packaging/macos/buildpackage.sh

```diff
@@ -127,8 +125,6 @@ modify_plist "{MINIMUM_SYSTEM_VERSION}" "10.11" "${TEMPLATE_DIR}/Contents/Info.p
 # Compile universal (x86_64 + arm64) arch-specific apphosts
 clang apphost.c -o "${TEMPLATE_DIR}/Contents/MacOS/apphost-x86_64" -framework AppKit -target x86_64-apple-macos10.15
 clang apphost.c -o "${TEMPLATE_DIR}/Contents/MacOS/apphost-arm64" -framework AppKit -target arm64-apple-macos10.15
-clang apphost-mono.c -o "${TEMPLATE_DIR}/Contents/MacOS/apphost-mono" -framework AppKit -target x86_64-apple-macos10.11
```

Comment:
The `MINIMUM_SYSTEM_VERSION` line just above will also need to be changed to the minimum version that .NET8 runs on (which may be older than the officially supported minimum of macOS 12).

Reply:
Confirmed that the net8 test build runs on 10.15, so `10.11` &rarr; `10.15` above.